studio_lenzing.css

```diff
@@ -0,0 +1,32 @@
+.navbar-default .navbar-brand,
+.navbar-default .navbar-brand:hover,
+.navbar-default .navbar-brand:focus {
+    color: black;
```

Comment:
Colors declared in hexadecimal (hex) are rendered faster than predefined colors. 

e.g.
black - #000

Reply:
My bad. Please look at this thread https://stackoverflow.com/questions/6797306/performance-differences-between-color-declarations

Reply:
Noted, thanks for this info.